src/migrations/r-migration-commands/m125_m126_no_click_animation_rename.ts

```diff
@@ -0,0 +1,31 @@
+import { RecordNode, r, rtp, RT, en } from "../../r";
+import { IOrder } from "../IOrder";
+
+
+/**
+ * assigns value opposite of no_click_animation to hover_animation as no_click_animation
+ * has a negative tone to it, confusing the user
+ */
+
+class Migration implements IOrder {
+  execute(projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.record(pJson);
+
+    projectF.getRecords(RT.scene).forEach(scene => {
+      const sceneF = r.record(scene);
+
+      //* all elements that have no_click_animation property in their props
+      const elements = sceneF.getAllDeepChildrenWithFilter(RT.element, e => e.props.hasOwnProperty(rtp.element.no_click_animation));
```

Comment:
you should not use hasOwnProperty. 
use `r.element(e).get(rtp.element.no_click_animation)` -> which should give you true/false/undefined

Add your logic based on it